editor/editor_node.cpp

```diff
@@ -3160,6 +3160,19 @@ void EditorNode::set_addon_plugin_enabled(const String &p_addon, bool p_enabled,
 		// Errors in the script cause the base_type to be an empty string.
 		if (String(script->get_instance_base_type()) == "") {
 			show_warning(vformat(TTR("Unable to load addon script from path: '%s' There seems to be an error in the code, please check the syntax."), script_path));
```

Comment:
It would be good to tell users about what is being done behind the scenes:
```suggestion
			show_warning(vformat(TTR("Unable to load addon script from path: '%s'. This might be due to an error in the code. Disabling the addon at '%s' to prevent further errors."), script_path, p_addon));
```